src/main/java/Awex.java

```diff
@@ -0,0 +1,130 @@
+import java.util.*;
+
+public class Awex {
+    public static void message() {
+        System.out.println("Input type must be one of:");
+        System.out.println("  1. list");
+        System.out.println("  2. mark <task number>");
+        System.out.println("  3. unmark <task number>");
+        System.out.println("  4. todo <task>");
+        System.out.println("  5. deadline <task> /by <deadline>");
+        System.out.println("  6. event <task> /from <start> /to <end>");
+        System.out.println("Type 'bye' to exit.");
+    }
+
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm AWEX!\nWhat can I do for you?");
+        LinkedList<Task> list = new LinkedList<>();
```

Comment:
Any reason why a LinkedList is used instead of an ArrayList? Indeed a LinkedList works similar to an ArrayList, although their use cases may vary depending on the scenario. Here is some research that I did: [ArrayList vs LinkedList](https://www.w3schools.com/java/java_linkedlist.asp)